bower.json

```diff
@@ -18,14 +18,13 @@
     "output"
   ],
   "dependencies": {
-    "purescript-prelude": "^4.0.0",
+    "purescript-prelude": "^4.1.0",
     "purescript-ring-modules": "^5.0.1",
-    "purescript-foreign-generic": "^7.0.0",
-    "purescript-simple-json": "^4.0.0",
-    "purescript-ordered-collections": "^1.0.0",
+    "purescript-foreign-generic": "^8.0.0",
+    "purescript-simple-json": "^4.3.0",
```

Comment:
We do not really need to update number in minor or patch part of version as we use `^`, `^x.a.b` means any version which has x as major and anything more then a.b is fine. if we used a new function which is added to purescript-simple-json 4.3 for example, which we don't.

Also this change is somewhat braking. This lib works fine with purescript-simple-json v4 as well as v4.1, v4.2, v4.3 etc, if this lib is installed in some lib/app where exact v4 of purescript-simple-json is used as dependency, build of that would fail. 

(Changing version of dependency is mostly a braking change.)

Basically we don't need to make change minor or patch part of versions where ^ is used unless the lib where we made major bump needs it (for example purescript-foreign-generic needs prelude ^4.1)